### PR TITLE
Basic streaming demo

### PR DESCRIPTION
**Pull request by rsowald**
PVC runs smoothly, button functions seem to be doing what they should, but getting a 409 ("conflict") error when running startStream.

```
{
    "jsonrpc": "2.0",
    "id": "3d888838-9748-4e77-95a4-371719ae7c7a",
    "method": "video.stream.start",
    "params": {
        "room_session_id": "c0aaa3e3-f672-4099-8981-8b292738fdf0",
        "url": "rtmp://a.rtmp.youtube.com/live2/0db0-kt6r-7b2h-bzf3-ce54"
    }
}



{
    "jsonrpc": "2.0",
    "id": "3d888838-9748-4e77-95a4-371719ae7c7a",
    "result": {
        "stream": null,
        "code": "409",
        "message": "Conflict"
    }
}

```
This is a recently created PVC on staging.

### CODE REVIEW

File: Video/README.md [hunk 1]
```diff
@@ -4,13 +4,16 @@
...
+Develop a simple full-stack video demo with this step-by-step walk-through.
 
 [Room Previews using Programmable Video Communication](./Room%20Preview%20Demo)  
-Very simple demonstration of getting and refreshing Room Previews.
+A very simple demonstration of getting and refreshing Room Previews.
 
 [Layout Positions](./Layout-Positions)  
 This guide shows how to control positions and layouts in a video room.
 
+[RTMP Streaming](./RTMP-Streaming/)
```
Comment (danieleds) [inline Video/README.md:15]: missing spaces at the end of the line

**Comment by danieleds** [comment]
can we add a .gitignore at least for node_modules?

**Comment by rsowald** [comment]
> can we add a .gitignore at least for node_modules?

oh. I added a monorepo-level gitignore. Probably not helpful for those who only want to clone one subrepo?

File: Video/README.md (hunk 1 above)
Comment (rsowald) [reply Video/README.md:15]: remind me one more time why we need 2 spaces at the end of the line?

Comment (danieleds) [reply Video/README.md]: to force a line break. Otherwise the line below goes next to the one above

**Review by danieleds — APPROVED** [review]

**Pull request merged**